src/lib.rs

```diff
@@ -1939,6 +1957,17 @@ impl RtcConfig {
         self.crypto_provider
     }
 
+    /// Returns the configured DTLS certificate configuration.
+    pub fn dtls_cert_config(&self) -> &DtlsCertConfig {
+        &self.dtls_cert_config
+    }
+
+    /// Set the DTLS certificate configuration for certificate generation.
```

Comment:
Let's flesh this out a bit. Show an example.

Reply:
Should I remove the set_dtls_cert()... it can be done using this same function and passing in DtlsCertConfig::PregeneratedCert instead of DtlsCertConfig::Options ? I kept it for API compat....same goes for get_dtls_cert().. you could just do get_dtls_cert_config() and that returns either Options or a PreeneratedCert